prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -46,6 +46,44 @@ items:
       - kind: ServiceAccount
         name: weave-net
         namespace: kube-system
+  - apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: Role
+    metadata:
+      name: weave-net2
```

Comment:
The existing one is a ClusterRole; this is a Role to narrow down the write permissions.

I agree with your point about naming; it was late...